stl/inc/vector

```diff
@@ -3424,6 +3424,155 @@ _NODISCARD _CONSTEXPR20 _Iter_diff_t<_VbIt> _Count_vbool(_VbIt _First, const _Vb
         });
 }
 
+template <class _VbIt, class _OutIt>
+_CONSTEXPR20 _OutIt _Copy_vbool(_VbIt _First, _VbIt _Last, _OutIt _Dest) {

```

Comment:
I'm questioning having this in the header, created #2462

Reply:
I am wondering whether we should make them static methods inside of vector<bool> itself

Reply:
I think method vs free function  and in fully header / mostly in .cpp are orthogonal questions.
I have no opinion on method vs free function.